packages/graphback-runtime/tests/data/RESTDataProviderTest.ts

```diff
@@ -0,0 +1,76 @@
+import _test, { TestInterface } from 'ava';
+import { buildSchema, GraphQLObjectType } from 'graphql';
+import { RESTDataProvider } from '../../src/data/RESTDataProvider';
+
+
+const schema = buildSchema(`
+"""
+@model
+"""
+type Users {
+ name: String!
+ job: String!
+
+}
+`);
+
+interface Context {
+    provider: RESTDataProvider;
+}
+
+const test = _test as TestInterface<Context>;
+const modelType = schema.getType('Users') as GraphQLObjectType
+
+test.beforeEach(async t => {
+    const provider = new RESTDataProvider(modelType,'https://reqres.in/api',{'Content-Type': 'application/json'});
+    t.context = { provider };
+});
+
+/**
+ * 
+ * Currently I just have 3 endpoints in my hosted server. So I tested only 
```

Comment:
SWAPI is standard for API. https://swapi.co/documentation
Let's use that

Reply:
Yeah I was gonna use that for testing. But they only provided the read operation. Thats why I used the other one.